examples/Cassie/urdf/cassie_v2.urdf

```diff
@@ -714,7 +714,7 @@
     <child link="shin_left" />
     <axis xyz="0 0 1" />
     <limit lower="-.1" upper="0.075"/>
-    <dynamics damping=".10" />
+    <dynamics damping="10" />
```

Comment:
I'm curious why the damping is 10?

Reply:
Will revert. These numbers, as far as I know, are not really based yet in empirical results.